rpmlb/builder/base.py

```diff
@@ -11,6 +12,17 @@
 LOG = logging.getLogger(__name__)
 
 
+#: Regular expression for finding macro definitions
+MACRO_REGEX = re.compile(
+    r'''^
+    %global\s+                   # beginning of definition
+    (?P<name>[^\s]+)\s+          # one-word name
+    (?P<value>(?:.|(?<=\\)\n)+)  # value including spaces and escaped newlines
+    $''',
+    flags=re.MULTILINE | re.VERBOSE,
+)
```

Comment:
I'm not sure about the regex definition on module level. Currently it is needed only by a single method, an I'm getting a bit of "global variable" feel from it. On the plus side, it is now importable and I can also use it in tests.

Suggestions?